lib/aws/rails/sqs_active_job/executor.rb

```diff
@@ -36,8 +42,31 @@ def execute(message)
               job_msg = job ? "#{job.id}[#{job.class_name}]" : 'unknown job'
               @logger.info "Error processing job #{job_msg}: #{e}"
               @logger.debug e.backtrace.join("\n")
+            ensure
+              refresh_monitor = false
             end
           end
+
+          @monitor.post(message) do |message|
+            while refresh_monitor
+              begin
+                # Extend the visibility timeout to the provided refresh timeout
+                message.change_visibility({ visibility_timeout: @visibility_refresh })
```

Comment:
is it more accurate to say update the `visibility_timeout` so the job is available to be picked up again after failure?